packages/eslint-config/README.md

```diff
@@ -0,0 +1,45 @@
+# v2 - wip
+
+# @bigcommerce/eslint-config
+
+This package is a configuration preset for [ESLint](https://eslint.org/).
+
+
+## Install
+
+```sh
+npm install --save-dev eslint prettier
+npm install --save-dev @bigcommerce/eslint-config
+```
+
+
+## Usage
+
+Add `@bigcommerce/eslint-config` to your project's ESLint configuration file. i.e.:
+
+```js
+// .eslintrc.js
+require('@bigcommerce/eslint-config/patch');
+
+module.exports = {
+  extends: ['@bigcommerce/eslint-config'],
+};
+```
+
+This config also runs prettier via eslint, add the following to your `package.json`
+```json
+{
+  "prettier": "@bigcommerce/eslint-config/prettier"
+}
+```
+
+If possible, try not to override the preset unless you have a special reason.
```

Comment:
🍹 I'm okay leaving this in, but I think this is more for our internal consumption. If any external developer using this package should be able to modify what they want out of our config - at least that's how I see it.